src/data/faq.json

```diff
@@ -336,6 +336,22 @@
+                            "These above mentioned displays give access to the following events:<ul><li>If the 3G rules are met, your certificates allow access to 3G events.</li><li>If 3G+ rules are met, your certificates allow access to 3G+ and 3G events.</li><li>When the 2G rules are met, your certificates allow access to 2G, 3G and 3G+ events.</li><li>When the 2G+ rules are met, your certificates allow access to 2G+, 2G, 3G and 3G+ events.</li></ul>",
+                            "<b>Why is the 2G+ status not displayed even though I have a vaccination or recovery certificate and a negative test?</b>",
+                            "The 2G+ status results from the combination of vaccination and test certificates. If the data within the certificates, for example name or date of birth, do not match, the Corona-Warn-App cannot assign these certificates to each other and determine the corresponding status. In this case, the Corona-Warn-App will show you two certificates in the overview, each of which is assigned the rules 3G and 2G. Therefore, make sure that your name is entered correctly and completely when issuing the certificate, and contact the issuer of the certificate if errors occured when entering the name or date of birth.",
+                            "A test certificate officially confirms a negative test result in the form of a QR code in the Corona-Warn-App. They can request the official test certificate for PCR and rapid tests when they register a test in the Corona-Warn-App. To do this, they can open the universal QR code scanner in the middle of the CWA tab and scan the QR code for test registration. They will receive this either while booking the test or on-site during testing, provided the provider is connected to the Corona-Warn-App. After scanning the QR code, a window will automatically open where they can request the test certificate. More information can be found in this blog: <a href='../blog/2021-06-24-cwa-version-2-4' target='_blank'>Project team releases rapid test partner search and Corona-Warn-App version 2.4 with digital test certificate</a>",
+                            "<b>Will 2G+ take my booster vaccination into account?</b>",
+                            "According to a decision by the federal and state health ministers, vaccinated or recovered persons who have received a booster vaccination are currently not required to submit a current test in the case of a 2G+ regulation. The Corona-Warn-App does not currently reflect this regulation in the status display. If a vaccination or recovery certificate and booster vaccination certificate are available, it will not display a 2G+ status, but will display 2G. The project team is already working on ensuring that the Corona-Warn-App will include certificates of booster vaccinations accordingly in the future.",

```

Comment:
```suggestion
                            "According to a decision by the federal and state health ministers, vaccinated or recovered persons who have received a booster vaccination are currently not required to submit a current test in the case of a 2G+ regulation. The Corona-Warn-App does not currently reflect this regulation in the status display. If a vaccination or recovery certificate and booster vaccination certificate are available, <b>it will not display a 2G+ status, but will display 2G</b>. The project team is already working on ensuring that the Corona-Warn-App will include certificates of booster vaccinations accordingly in the future.",
```